### PR TITLE
[NO SYNTAX CHANGES] Fix argument name mismatch warning and shadowing local variable

### PR DESCRIPTION
**Pull request by gAlfonso-bit**
No syntax other than variable names have been changed.

## **Discord contact info**

AlfsonsoB#1500

### CODE REVIEW

**Comment by gAlfonso-bit** [comment]
If there’s a problem with the variable names, please do not hesitate to say so.

File: src/battle_interface.c [hunk 1]
```diff
@@ -2340,11 +2340,11 @@ static s32 CalcNewBarValue(s32 maxValue, s32 oldValue, s32 receivedValue, s32 *c
 
     if (maxValue < scale) // handle cases of max var having less pixels than the whole bar
     {
-        s32 toAdd = Q_24_8(maxValue) / scale;
```
Comment (huderlem) [inline src/battle_interface.c:2343]: I think keeping the variable shadowing might be fine here.  Changing it to a Frankenstein name isn't worth it, in my opinion.  `delta` might be a better name than `toAdd` for the whole function..

**Comment by gAlfonso-bit** [comment]
Does this work or should I just remove the change and only keep the Battle Factory function change

**Comment by huderlem** [comment]
> Does this work or should I just remove the change and only keep the Battle Factory function change

No, that is not what I meant.  I don't think this PR adds any value as a whole--changing the entire function to use a "delta" instead of "toAdd" name isn't really an improvement either.

**Pull request closed without merging**